src/recipe.rst

```diff
@@ -66,17 +66,20 @@ conda-forge.
 
 Optional: ``bld.bat`` and/or ``build.sh``
 ------------------------------------------
-In many cases, ``bld.bat`` and/or ``build.sh`` files are not required. Pure Python packages almost never need it.
+In many cases, ``bld.bat`` and/or ``build.sh`` files are not required. Pure Python packages almost never need them.
 If the build can be executed with one line, you may put this line in the ``script`` entry of the ``build`` section of
-the ``meta.yaml`` file.
+the ``meta.yaml`` file. (For instance, if a file uses `setuptools`, try ``script: python setup.py install --single-version-externally-managed --record=record.txt``.)
```

Comment:
I believe that is "kind of" implicit there in the phrase "if a file uses `setuptools`." However, I would change file to package.
Also, being explicit is not bad too.

@pmlandwehr if you have the time for those changes let me know. I can merge this and do that later.